cluster-api/src/main/resources/application.properties

```diff
@@ -45,24 +61,22 @@ klaw.retries.config=10
 klaw.retry.backoff.ms=5000
 klaw.request.timeout.ms=15000
 
+# default Kafka SASL properties
+kafkasasl.saslmechanism.plain=PLAIN
+kafkasasl.saslmechanism.gssapi=GSSAPI
+kafkasasl.saslmechanism.scram.256=SCRAM-SHA-256
+kafkasasl.saslmechanism.scram.512=SCRAM-SHA-512
+
 # swagger documentation path parser
 spring.mvc.pathmatch.matching-strategy = ANT_PATH_MATCHER
 
 spring.banner.location=classpath:banner.txt
+# log file settings
+logging.file.name=./../logs/kw-clusterapi.log
+spring.mvc.log-resolved-exception=true
+logging.level.root=info
 
-# custom acls api endpoints
+# Custom Acls - Api Endpoints (Currently applicable on Aiven calls)
 klaw.clusters.listacls.api=https://api.aiven.io/v1/project/projectName/service/serviceName/acl
 klaw.clusters.addacls.api=https://api.aiven.io/v1/project/projectName/service/serviceName/acl
-klaw.clusters.deleteacls.api=https://api.aiven.io/v1/project/projectName/service/serviceName/acl/aclId
-
-# access token for https requests
-klaw.clusters.accesstoken=
-
-#aiven schema security
-klaw.aiven.karapace.credentials=
-klaw.aiven.kafkaconnect.credentials=
-
-# log file settings
-logging.level.root=info
-logging.file.name=./../logs/kw-clusterapi.log
-spring.mvc.log-resolved-exception=true
\ No newline at end of file
+klaw.clusters.deleteacls.api=https://api.aiven.io/v1/project/projectName/service/serviceName/acl/aclId
```

Comment:
Nitpick, missing empty break line

Reply:
Sure, will include this change in next PR.